### ISSUE TITLE
Cannot call read_image

### ISSUE THREAD

**Issue by muff1nman** [openshift/openshift-restclient-python#191]
When trying to use read_image with python3 with the change in #187, the following exception is occurring:

```
Traceback (most recent call last):
  File "test.py", line 51, in <module>
    image_data = image_api.read_image(image_sha)
  File "/opt/app-root/src/venv/lib/python3.5/site-packages/openshift/client/apis/image_openshift_io_v1_api.py", line 2972, in read_image
    (data) = self.read_image_with_http_info(name, **kwargs)
  File "/opt/app-root/src/venv/lib/python3.5/site-packages/openshift/client/apis/image_openshift_io_v1_api.py", line 3057, in read_image_with_http_info
    collection_formats=collection_formats)
  File "/opt/app-root/src/venv/lib/python3.5/site-packages/kubernetes/client/api_client.py", line 321, in call_api
    _return_http_data_only, collection_formats, _preload_content, _request_timeout)
  File "/opt/app-root/src/venv/lib/python3.5/site-packages/kubernetes/client/api_client.py", line 163, in __call_api
    return_data = self.deserialize(response_data, response_type)
  File "/opt/app-root/src/venv/lib/python3.5/site-packages/kubernetes/client/api_client.py", line 236, in deserialize
    return self.__deserialize(data, response_type)
  File "/opt/app-root/src/venv/lib/python3.5/site-packages/openshift/client/api_client.py", line 22, in _ApiClient__deserialize
    return super(ApiClient, self).__deserialize_model(data, klass)
  File "/opt/app-root/src/venv/lib/python3.5/site-packages/kubernetes/client/api_client.py", line 620, in __deserialize_model
    kwargs[attr] = self.__deserialize(value, attr_type)
  File "/opt/app-root/src/venv/lib/python3.5/site-packages/openshift/client/api_client.py", line 16, in _ApiClient__deserialize
    return super(ApiClient, self).__deserialize(data, klass)
  File "/opt/app-root/src/venv/lib/python3.5/site-packages/kubernetes/client/api_client.py", line 276, in __deserialize
    return self.__deserialize_model(data, klass)
  File "/opt/app-root/src/venv/lib/python3.5/site-packages/kubernetes/client/api_client.py", line 622, in __deserialize_model
    instance = klass(**kwargs)
  File "/opt/app-root/src/venv/lib/python3.5/site-packages/kubernetes/client/models/runtime_raw_extension.py", line 49, in __init__
    self.raw = raw
  File "/opt/app-root/src/venv/lib/python3.5/site-packages/kubernetes/client/models/runtime_raw_extension.py", line 75, in raw
    raise ValueError("Invalid value for `raw`, must be a follow pattern or equal to `/^(?:[A-Za-z0-9+\/]{4})*(?:[A-Za-z0-9+\/]{2}==|[A-Za-z0-9+\/]{3}=)?$/`")
ValueError: Invalid value for `raw`, must be a follow pattern or equal to `/^(?:[A-Za-z0-9+\/]{4})*(?:[A-Za-z0-9+\/]{2}==|[A-Za-z0-9+\/]{3}=)?$/`
```

Via the debugger it looks like it may be tripping on the fact the input is not str but bytes.

**Comment by muff1nman** [comment]
Might need to have this line instead:

    data = {'Raw': b64encode(json.dumps(data).encode()).decode()}

**Comment by fabianvf** [comment]
Closed by #191, thanks @muff1nman
